firebase-docs.md

```diff
@@ -0,0 +1,248 @@
+      "apiKey": "jfdskjlksdflskfkslfk-QEhUe_bI",
+      "authDomain": "codemark-hfksjdfksdf.firebaseapp.com",
+      'databaseURL': dbURL,
+      "projectId": "codemark-6f33a",
+      "storageBucket": "codemark-sdjfkjkflsd.appspot.com",
+      "messagingSenderId": "jfdskfjdsk",
+      "appId": "1:lkjsdfjsdjfklsj:web:hsdfkjkdlsjfklsdj",
+      "serviceAccount": SERVICEACCOUNTFILE
+   }
+   STORAGE_BUCKET_URL = 'codemark-jkhsdkfjskdfjlksdjf.appspot.com'
+
+   # from web3.storage
+   web3storage_api_key = 'zxjkkcljzxkc.eyJzdWIiOiJkaWQ'
+
+   ```
+   <b>Note:</b> You will need to replace the `api_key`, `web3storage_api_key` in your `secrets.py` file with your own `OPENAI_API_KEY`, and the `web3storage_api_key` value in the file.
+
+
+### Import JSON Data Using the GUI
+
+If you prefer a graphical interface for importing data into Firebase, follow these steps to use the Firebase Console GUI:
+
+1. **Go to the Firebase Console**: Navigate to the [Firebase Console](https://console.firebase.google.com/).
+
+2. **Select Your Project**: Click on your Firebase project in the console. If you haven't created a project yet, follow the earlier instructions to create one.
+
+3. **Access Realtime Database**: In the Firebase Console, click on the **Realtime Database** tab on the left sidebar.
+
+4. **Import Data**: Click on the **Import JSON** button located at the top of the Realtime Database section.
+
+5. **Choose Your JSON File**: A file selection dialog will appear. Select the `Schemas/dummy_data.json` file from your project directory.
+
+6. **Start Import**: After selecting the file, click on the **Open** or **Import** button (the label may vary depending on your operating system).
+
+7. **Confirm Import**: Firebase will display a preview of the data to be imported. Verify that it matches your dummy data structure.
+
+8. **Initiate Import**: If everything looks correct, click on the **Import** button to start the data import process.
+
+9. **Monitor Progress**: Firebase will show the progress of the data import. Once completed, you will receive a confirmation message.
+
+
+
+<details><summary>Optional: <b>Uploading through a Python script</b></summary>
+
+**If you prefer to upload data using a Python script, follow these steps:** creating a new Python file called `push_dummy_data.py` and adding the following code:
+
+   ```python
+
+   ### Optional: Uploading Data Through a Python Script
+
+   import firebase_admin
+   from firebase_admin import credentials
+   from firebase_admin import db
+   import json
+
+   # Initialize Firebase Admin SDK with your service account credentials
+   cred = credentials.Certificate("firebase/res/service-account-file.json")
+   firebase_admin.initialize_app(cred, {
+       'databaseURL': 'https://your-firebase-project.firebaseio.com'
+   })
+
+   # Reference to the Firebase Realtime Database
+   ref = db.reference("/")
+
+   # Load the dummy data from the schema file
+   with open("schemas/dummy_data.json", "r") as file:
```

Comment:
Some minor fixes the path in the open should be `"../Schemas/dummy_data.json"`, presuming push_dummy_data.py is in codemark directory (please explicitly, mention path of code at top of code or in instruction)